owl2nl/src/main/java/org/aksw/owl2nl/OptimizerDepParse.java

```diff
@@ -0,0 +1,239 @@
+package org.aksw.owl2nl;
+
+import edu.stanford.nlp.ling.IndexedWord;
+import edu.stanford.nlp.pipeline.CoreDocument;
+import edu.stanford.nlp.pipeline.CoreSentence;
+import edu.stanford.nlp.pipeline.StanfordCoreNLP;
+import edu.stanford.nlp.semgraph.SemanticGraph;
+import edu.stanford.nlp.semgraph.SemanticGraphFormatter;
+//import org.graalvm.compiler.core.common.type.ArithmeticOpTable;
+
+import javax.xml.soap.SOAPPart;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+public class OptimizerDepParse {
+
+    public String optimize(String text) {
+        try {
+            if (text == null || text == "") return text;
+
+            //text="something that a man that sings rock and that a man that sings jazz or a man that sings karaoke  or a man that sings bal";
```

Comment:
Same as the previous comment.

Reply:
For now need it for demonstration purposes. I will remove it afterwards. Thank you